src/Robo/Commands/Setup/ConfigCommand.php

```diff
@@ -39,8 +39,23 @@ public function import() {
       // If using core-only or config-split strategies, first check to see if
       // required config is exported.
       if (in_array($strategy, ['core-only', 'config-split'])) {
-        $core_config_file = $this->getConfigValue('docroot') . '/' . $this->getConfigValue("cm.core.dirs.$cm_core_key.path") . '/core.extension.yml';
+        // Read the config from the default folder (/config/default). This will
+        // Also be the fallback option, so some domains can still use default.
+        $core_config_file = $this->getConfigValue('docroot') . '/' .
+          $this->getConfigValue("cm.core.dirs.$cm_core_key.path") .
+          '/core.extension.yml';
+
+        // We don't want configs to be shared between multisites.
+        if ($this->getConfigValue('cm.share-configs') == FALSE) {
+          $drush_uri = $this->getConfigValue('drush.uri');
```

Comment:
Let's use the `site` here rather than the Drush URI, so that it matches the directory structure in `docroot/sites`:
`$this->getConfigValue('site')`

Every BLT command operates within a specific site context. This is 'default' by default, but can be modified at run-time by passing the `site` parameter.

Reply:
@alex-moreno actually is such a large code change here even necessary? I think the functionality you desire is already supported. You should be able to just set the following parameter in each site's `site.yml`:
`cm.core.dirs.$cm_core_key.path` (or specifically maybe `cm.core.dirs.sync.path`)